.github/workflows/integration-test.yml

```diff
@@ -118,7 +112,14 @@ jobs:
         env:
           EARTHDATA_USERNAME: ${{ secrets.EDL_USERNAME }}
           EARTHDATA_PASSWORD: ${{ secrets.EDL_PASSWORD }}
-        run: nox -s integration-tests -- --cov=earthaccess --cov-report=term-missing --capture=no --tb=native --log-cli-level=INFO
+        run: |
+          nox -s integration-tests -- \
+            --python=${{ matrix.python-version }} \
```

Comment:
`--python` is a nox option, not a pytest option, so it should come before the ` -- `

Reply:
Yeah, mistake :)